test/components/resizable_test.dart

```diff
@@ -21,27 +19,26 @@ Vector2 size = Vector2(1.0, 1.0);
 
 void main() {
   group('resizable test', () {
-    test('propagate resize to children', () {
```

Comment:
Why remove this test?

Reply:
Because resizable doesn't have children. Position component has.

Reply:
But this doesn't have to be applied on a position component right?
Shouldn't it be kept in here, and removed from the PositionComponent?

Reply:
Not exacly. 

Pre v1 refactor we had this mixin with no clear role. It was doing twp things: keeping a reference to a (game)size and calling resize in child components (for composed component).

This leads to the misconception that onlt `Resizable` components have the `resize` (now `onGameResize`) method, which is not true. All components have it.

Reply:
In another words: Resizable is not responsible for calling `onGameResize` and never was.

Reply:
Yeah, now I understand, this will be much more clear to that the responsibility is on the components with children.